_data/bgargi/bgargi.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Gargi Balasubramaniam
+institution: BITS Pilani, Goa
+profile_pic: gb.png # Name and extension of your profile picture(ex. mona.png)
```

Comment:
Hi @bgargi, I noticed that the extension here and the one in your image don't match. In order to make it easier to merge your PR, I would suggest you fix this issue

Reply:
Hi! Done.